components/cards/gifts/components/add-to-wishlist-form.tsx

```diff
@@ -14,25 +13,30 @@ type AddToWishListFormProps = {
 
 function AddToWishListForm({ giftId, wishlistId }: AddToWishListFormProps) {
   const { toast } = useToast();
-  const router = useRouter();
 
   const handleAddGiftToWishList = async (formData: FormData) => {
     if (!wishlistId) {
-      router.push('/register');
```

Comment:
Por qué se cambió acá ?

Reply:
just speed thing, with window.location.href is faster